docker-compose-starburst.yml

```diff
@@ -15,6 +15,45 @@ services:
     environment:
       POSTGRES_USER: dbt-trino
       POSTGRES_PASSWORD: dbt-trino
+    profiles:
+        - postgres
```

Comment:
Does it disable postgres container from running by default?

Reply:
Yes, I had to upgrade the docker-compose version to 3.9 and add `export COMPOSE_PROFILES=datalake`. 

```
Creating minio ... 
Creating dbt-trino_trino_1 ... 
Creating dbt-trino_hive-metastore_1 ... 
Creating minio                      ... done
Creating dbt-trino_mc-job_1         ... 
Creating dbt-trino_hive-metastore_1 ... done
Creating dbt-trino_trino_1          ... done
Creating dbt-trino_mc-job_1         ... done
```